helm_deploy/values-dev.yaml

```diff
@@ -38,6 +38,7 @@ generic-service:
     PRISONER_SEARCH_API_URL: https://prisoner-search-dev.prison.service.justice.gov.uk
     LOCATIONS_INSIDE_PRISON_API_URL: https://locations-inside-prison-api-dev.hmpps.service.justice.gov.uk
     HEALTH_AND_MEDICATION_API_URL: "https://health-and-medication-api-dev.hmpps.service.justice.gov.uk"
+    GOTENBERG_API_URL: 'http://hmpps-digital-prison-services-gotenberg'
```

Comment:
Given this is the same across all envs, it could go in the values.yaml file instead